.github/action-common-python-release/action.yml

```diff
@@ -0,0 +1,72 @@
+name: "Base Python Release"
+description: "Common tasks required to release python wheels on any platform"
+
+inputs:
+  package-name:
+    required: true
+    description: "Either `qcs-sdk-python` or `qcs-sdk-python-grpc-web`"
+
+  python-version:
+    default: "3.12"
+    description: "Python version used for script steps"
+  python-architecture:
+    description: "Python architecture used for script steps"
+  rust-target:
+    description: "Rust target used for script steps"
+  maturin-command:
+    default: build
+    description: "Maturin command used, `build` or `sdist`"
+  maturin-args:
+    description: "Additional maturin command arguments"
+  maturin-target:
+    description: "Rust target used for maturin compilation"
+
+
+runs:
+  using: "composite"
+  steps:
+  - uses: actions/setup-python@v4
+    with:
+      python-version: ${{ inputs.python-version }}
+      architecture: ${{ inputs.python-architecture }} 
+  - name: Install Rust toolchain 
+    uses: actions-rs/toolchain@v1 
+    with: 
+      toolchain: stable 
+      target: ${{ inputs.rust-target }}
+      profile: minimal 
+      default: true 
+  - name: Patch package metadata for grpc-web
+    shell: bash
+    if: matrix.package-name == 'qcs-sdk-python-grpc-web'
+    run: |
+      pip install toml
+      python crates/python/scripts/patch_grpc_web.py
+      cargo update
+
+  # must be build or sdist
+  - if: inputs.maturin-command == 'build'
+    name: Maturin - Build
+    uses: messense/maturin-action@v1
+    with:
+      command: ${{ inputs.maturin-command }}
+      target: ${{ inputs.maturin-target }}
+      args: -i 3.8 3.9 3.10 3.11 3.12 --release --manifest-path crates/python/Cargo.toml --out dist ${{ inputs.package-name == 'qcs-sdk-python-grpc-web' && '-F grpc-web' || '' }}
```

Comment:
???

```suggestion
      args: -i ${{ inputs.python-version }} --release --manifest-path crates/python/Cargo.toml --out dist ${{ inputs.package-name == 'qcs-sdk-python-grpc-web' && '-F grpc-web' || '' }}
```

Reply:
The `-i` argument uses one command to build all the variants, instead of creating a separate matrix entry for each version. Seems a little bit less resource intensive this way, unless you think the previous way is better?

Reply:
These are the contents in the `qcs-sdk-python-grpc-web` artifacts, juts to prove that all python versions are being built

<img width="709" alt="image" src="https://github.com/rigetti/qcs-sdk-rust/assets/97701976/8e4802a4-1bee-476f-a2ac-fe6c9451a832">

I'm going to resolve this comment, but please feel free to unresolve.

Reply:
Good point - removed `input.python-version`